src/js/visual/configurator-csr.js

```diff
@@ -1740,8 +1740,7 @@ ripe.ConfiguratorCsr.prototype._onPostConfig = async function(self, config) {
         await this.flushPending(true);
 
         self.loading = false;
-        this.trigger("ready");
-        this.trigger("post_config", config);
+        this.trigger("ready", { id: "configurator_csr" });

```

Comment:
I suggest allowing to specify the ID as an optional parameter here: https://github.com/ripe-tech/ripe-sdk/blob/3bf809a5a6fee6d905fbfc007a53d81cd8f639cc/src/js/visual/configurator-csr.js#L60

This way it will make it possible to differantiate the configurators. Maybe also add to PRC configurator?